engine/src/main/scala/cromwell/webservice/metadata/MetadataBuilderActor.scala

```diff
@@ -163,7 +163,12 @@ object MetadataBuilderActor {
   private def parseWorkflowEventsToTimestampedJsValue(events: Seq[MetadataEvent], includeCallsIfEmpty: Boolean): JsObject = {
     // Partition if sequence of events in a pair of (Workflow level events, Call level events)
     val (workflowLevel, callLevel) = events partition { _.key.jobKey.isEmpty }
-    val foldedWorkflowValues = eventsToIndexedJson(reduceWorkflowEvents(workflowLevel)).toJson.asJsObject
+    val reducedEvents = reduceWorkflowEvents(workflowLevel)
+    val reducedFields = if (reducedEvents.isEmpty) {
+      Map.empty
+    } else {
```

Comment:
Is this `if` actually required?  I would have thought the `suml` magic in `eventsToIndexedJson` would handle the case of an empty `reducedEvents` list correctly.